### ISSUE TITLE
API status doesn't change when the api goes down

### ISSUE THREAD

**Issue by geniusit** [katiawheeler/healthy#11]
Hi,

It seems that status are always in Operational state even if the API goes down ...
Could you have a look please ?
Tested with last create-react-app ...

Thank you

**Comment by geniusit** [comment]
It seems not really an issue but a design

**Comment by katiawheeler** [comment]
@geniusit were you able to figure it out? I'd be happy to take a look if anything needs to be addressed.

**Comment by geniusit** [comment]
@katiawheeler currently you check if the API returns a response != ok to consider a service interruption.
But if the API goes down you catch it and console.log('unhandled')
But the problem is that the status on the GUI stays 'Operational'. I think the GUI should manage the case when the API doesn't answer. What do you think about it ?

**Comment by katiawheeler** [comment]
@geniusit Sure I can update this

**Comment by geniusit** [comment]
I m trying to implement it. When the API goes down the status is correctly updated. But from down to up I need to manually refresh the page to see the Operational status.
I m looking a way to : 

```javascript
this.setState(
      {
        apis,
        hasError: false
      }
    );
```

I don't understand why this onResponse is always undefined (onResponse || undefined)

```javascript
if (onResponse) {
    onResponse(api, apiResponse);
  }
```

I don't understand how this method is called : 

```javascript
public handleError = (api: Api, response: Response) => {
    console.log('handle error');
    const apis = [...this.state.apis];
    const index = apis.findIndex(stateApi => stateApi.api === api);
    apis[index].hasError = true;
    apis[index].response = response;

    this.setState(
      {
        apis,
        hasError: true
      }
    );
  };
```
Could you help me ?

**Comment by geniusit** [comment]
I have done some modification and it works in both ways now.
Is it possible to let me create a branch + PR ? 
Thank you

**Comment by katiawheeler** [comment]
@geniusit you should be able to fork the repo, open a branch against it, and push. You should find instructions here: https://opensource.com/article/19/7/create-pull-request-github

**Comment by katiawheeler** [comment]
This should be fixed with version 2.0.0.
